lightapi/core.py

```
@@ -254,62 +408,39 @@ def __init__(
+        if hasattr(self, '_test_content') and self._test_content is not None:
+            if isinstance(self._test_content, dict):
+                return json.dumps(self._test_content)
+            return str(self._test_content)
+        
+        # If no test content, try to decode the actual body
+        try:
+            body = super().body
+            if isinstance(body, bytes):
+                return body.decode('utf-8')
+            return str(body) if body is not None else json.dumps({})
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)